corehq/apps/geospatial/const.py

```diff
@@ -1,5 +1,5 @@
 
-GEO_POINT_CASE_PROPERTY = 'commcare_gps_point'
+GEO_POINT_CASE_PROPERTY = 'gps_point'
```

Comment:
nit: would be nice to either rename the constant to GPS_POINT_CASE_PROPERTY or update the value to be `geo_point`

Reply:
Renamed to `geo_point` ([afa1dcb](https://github.com/dimagi/commcare-hq/pull/33420/commits/afa1dcbe806d736eee94839d5ceb73776430bb13)).

Reply:
Sorry, going back on this.
I think having gps in the name is important since that is the term really used for GPS coordinates, so `gps_point` will make it explicit what the case property is referring to than `geo_point`. So can we rename the constant instead?
Even data dictionary, calls the datatype GPS.

If you are using an IDE it should be easy to just replace all usages of the constant, I see 8 usages in total, including the constant declaration.

Reply:
[28059fb](https://github.com/dimagi/commcare-hq/pull/33420/commits/28059fb0a130082c1bdb342c104845c5bb2eb5fa)